lucien-ios/Dashboard/DashboardCollectionView/DashboardCollectionViewModel.swift

```diff
@@ -31,6 +31,10 @@ final class DashboardCollectionViewModel {
     func getComicCount() -> Int {
         return dashboardComics.count
     }
+        
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)